ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/ContentFileUtil.java

```diff
@@ -22,12 +22,34 @@
 import com.netease.arctic.data.DataFileType;
 import com.netease.arctic.data.file.ContentFileWithSequence;
 import com.netease.arctic.data.file.WrapFileWithSequenceNumberHelper;
+import com.netease.arctic.table.UnkeyedTable;
+import com.netease.arctic.utils.ManifestEntryFields;
+import com.netease.arctic.utils.TableFileUtils;
 import org.apache.iceberg.ContentFile;
+import org.apache.iceberg.DataFile;
 import org.apache.iceberg.DataFiles;
 import org.apache.iceberg.FileMetadata;
+import org.apache.iceberg.HasTableOperations;
+import org.apache.iceberg.MetadataTableType;
+import org.apache.iceberg.MetadataTableUtils;
 import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.data.GenericRecord;
+import org.apache.iceberg.data.IcebergGenerics;
+import org.apache.iceberg.data.Record;
+import org.apache.iceberg.io.CloseableIterable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Tools for handling the ContentFile which in Iceberg
+ */
 public class ContentFileUtil {
```

Comment:
It may not be a good idea to put `getAllContentFilePath()` into `ContentFileUtil`.

`UnKeyedTableUtil` may be a better choice, because it seems to be an operation on `Table` as opposed to `File`.